MiniScript-cs/MiniscriptTypes.cs

```diff
@@ -66,14 +66,6 @@ public abstract class Value {
 			return (int)DoubleValue();
 		}
 		
-		/// <summary>
-		/// Get the numeric value of this Value as an unsigned integer.
-		/// </summary>
-		/// <returns>this value, as unsigned int</returns>
-		public virtual uint UIntValue() {
-			return (uint)DoubleValue();
-		}
-		
```

Comment:
This doesn't have anything to do with Error constructors, why is deletion of this method required?